sources/web/datalab/polymer/modules/file-manager-factory/file-manager-factory.ts

```diff
@@ -22,67 +22,65 @@ enum FileManagerType {
 }
 
 interface FileManagerConfig {
-  className: any;
+  className: new () => FileManager;
```

Comment:
This field is still not a name, but a class. Maybe call it managerClass?

Reply:
Done, renamed to `class`.